TestSuite/CMakeLists.txt

```diff
@@ -3,11 +3,50 @@ quicc_add_benchmark(Explicit
   MODEL "BoussinesqShellRTC"
   WORKDIR "${CMAKE_BINARY_DIR}/${QUICC_CURRENT_MODEL_DIR}/TestSuite/Benchmarks"
   ARCHIVEDIR "${CMAKE_BINARY_DIR}/Models/_refdata"
+  VARIANTS "framework/parallel/algorithm:single1d"
+  MPIRANKS 6
+  )
+
+quicc_add_benchmark(Explicit
+  MODEL "BoussinesqShellRTC"
+  WORKDIR "${CMAKE_BINARY_DIR}/${QUICC_CURRENT_MODEL_DIR}/TestSuite/Benchmarks"
+  ARCHIVEDIR "${CMAKE_BINARY_DIR}/Models/_refdata"
+  RUNID "_single1d"
+  VARIANTS "framework/parallel/algorithm:single1d"
+  MPIRANKS 6
+  )
+
+quicc_add_benchmark(Explicit
+  MODEL "BoussinesqShellRTC"
+  WORKDIR "${CMAKE_BINARY_DIR}/${QUICC_CURRENT_MODEL_DIR}/TestSuite/Benchmarks"
+  ARCHIVEDIR "${CMAKE_BINARY_DIR}/Models/_refdata"
+  RUNID "_single2d"
```

Comment:
can't this info be extracted by the `VARIANTS` variable?

Reply:
Is it better to keep in independent? RUNID is just an arbitrary choice. Otherwise I would need to append all changes to the name.

Reply:
Eventually we will run something like: triangula truncation, tubular splitting, equation grouping. But do we want to name it Balallb_tubular_equation_triangular?

Reply:
uhmm, I don't know the ID seems redundant to me, and having it programmatically depend on the options with some convention it seems easier to implement and not mess up

Reply:
It now has a very naive implementation and creates a RUNID based on the VARIANTS. We will need to update when more complicated configuration is needed. the ID needs to be copied into the CI YML generator script anyways...

Reply:
But at least now it is controlled by the CMake function, and we can work on that without modifying the model setup.